student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
```

Comment:
* AS stated in the [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Constructor%20extends%20parent%27s%20constructor%20by%20adding%20%40classroom%20and%20a%20parameter%20for%20it.) `Constructor extends parent's constructor by adding @classroom and a parameter for it.` for example
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```